etc/databases/world/updates/updates.sql

```diff
@@ -30417,5 +30417,29 @@ begin not atomic
 
         insert into applied_updates values ('050320231');
     end if;
+
+    -- 06/03/2023 2
+    if (select count(*) from applied_updates where id = '060320232') = 0 then
+        -- Update name requirements and quest text for quest 951 to what it was like during alpha.
+        update quest_template set Title = 'Blackfathom Relics', ReqItemId2 = 5330, ReqItemId3 = 5331, ReqItemCount1 = 2, ReqItemCount2 = 2, ReqItemCount3 = 2, Objectives = 'Bring 2 Blackfathom Relics, 2 Mathystra Relics and 2 Elven Cup Relics to Onu at the Grove of the Ancients.', OfferRewardText = 'Thank you. These relics are from a time when Mathystra shone. That once great bastion of the elves has faded, but shards of its magic remain. Let us hope we can unlock the secrets of that place before our enemies...$B$BBe vigilant, $N. And prepare yourself to return to the deeps within the Ruins of Mathystra. Fate will, one day, take you there.' where entry = 951;
+
+        -- Reactivate Blackfathom Relic (5331).
+        update item_template set name = '[PH] Blackfathom Relic', display_id = 8290, stackable = 2, max_count = 2, bonding = 2, flags = 0, buy_price = 0, sell_price = 0 where entry = 5331;
```

Comment:
`bonding` should be set to `4` which is `Quest Item` according to https://archive.thealphaproject.eu/media/Alpha-Project-Archive/Images/Azeroth/Kalimdor/Darkshore/images_4335.jpg

Reply:
Bonding fixed.